muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/containers/main-function.tsx

```diff
@@ -131,6 +130,7 @@ interface MainFunctionProps {
   websocket: Websocket;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 (window as any).USES_HISTORY_API = true;
```

Comment:
You can get rid of the any with extending  the window Ambient type

```
declare global {
    interface Window { 
         USES_HISTORY_API: true
    }
}
window.USES_HISTORY_API = true
```

Reply:
Is this really necessary to fix in this branch? Outside side of whole branch scope anyways. Can be done later with separate feature branch